src/components/SurveyAnswer/ChoiceAnswer/MultiplePickAnswerItem.tsx

```diff
@@ -0,0 +1,51 @@
+import { useState } from 'react';
+import Image from 'next/image';
+
+import { SurveyAnswer } from 'services/surveyInterfaces';
+
+import styles from './ChoiceAnswer.module.scss';
+
+interface MultiplePickAnswerItemProps {
+  answer: SurveyAnswer;
+  onAnswerItemSelect: (answerId: string) => void;
+}
+
+const MultiplePickAnswerItem = ({
+  answer,
+  onAnswerItemSelect,
+}: MultiplePickAnswerItemProps) => {
+  const [isSelectedAnswer, setIsSelectedAnswer] = useState(false);
+
+  const answerStateClass = () => {
+    if (isSelectedAnswer) {
+      return styles.activeAnswer;
+    }
+  };
+
+  const onAnswerItemClick = () => {
+    setIsSelectedAnswer(!isSelectedAnswer);
+    onAnswerItemSelect(answer.id);
+  };
+
+  return (
+    <div
+      key={answer.id}
```

Comment:
As you have the `key` attribute already set on the "instantiation" of this component (in the parent component), I think you do not need it inside the component ;-) 

```suggestion
```